carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/decorator/ExtendedWebElement.java

```diff
@@ -318,24 +319,34 @@ public boolean isPresent(By by, long timeout) {
      */
 	private boolean waitUntil(ExpectedCondition<?> condition, long timeout) {
 		boolean result;
+		originalException = null;
 		
 		final WebDriver drv = getDriver();
 		
 		Timer.start(ACTION_NAME.WAIT);
 		wait = new WebDriverWait(drv, timeout, RETRY_TIME).ignoring(WebDriverException.class)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.53.207.202/sonarqube/coding_rules#rule_key=squid%3AS2696)